plugins/packages/amazonses/lib/manifest.json

```diff
@@ -0,0 +1,151 @@
+
+{
+  "$schema": "https://raw.githubusercontent.com/ToolJet/ToolJet/develop/plugins/schemas/manifest.schema.json",
+  "title": "Amazon SES datasource",
+  "description": "A schema defining Amazon SES API",
+  "type": "api",
+  "source": {
+    "name": "Amazon SES",
+    "kind": "amazon-ses",
```

Comment:
please change it to `amazonses`, the directory name `plugins/packages/amazonses` and kind should match